src/NLog.Web.AspNetCore/NLogRequestPostedBodyMiddlewareConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+
+namespace NLog.Web
+{
+    /// <summary>
+    /// Contains the configuration for the NLogRequestPostedBodyMiddleware
+    /// </summary>
+    public class NLogRequestPostedBodyMiddlewareConfiguration
+    {
+        /// <summary>
+        /// The default configuration
+        /// </summary>
+        public static readonly NLogRequestPostedBodyMiddlewareConfiguration Default = new NLogRequestPostedBodyMiddlewareConfiguration();
+
+        /// <summary>
+        /// Defaults to true
+        /// </summary>
+        public bool DetectEncodingFromByteOrderMark { get; set; } = true;
```

Comment:
Please remove this as setting, until a user needs to change it.

Reply:
Complete